helpers/scripts/registerServer.js

```diff
@@ -0,0 +1,48 @@
+const fetch = require('node-fetch');
+
+const AUTH_URL = 'http://819e7f03.ngrok.io';
+const SERVER_NAME = 'Test Server';
+const SERVER_URL = 'http://70e8a7cf.ngrok.io';
```

Comment:
Could these URLs just default to `localhost:<default-port>` so you don't need to use ngrok?

Reply:
They could I guess. These were more just sample values. Keep in mind that eduroam doesn't like localhost much

Reply:
I'm just thinking if this is localhost then someone using the script wouldn't have to edit it, assuming they have both servers running locally. Eduroam shouldn't matter here since it's all local I'm assuming

Reply:
The server may be able to communicate locally with the auth-server, but once a client (admin, web, or mobile) is added into the mix everything needs to be ngrok. Maybe `AUTH_URL` could be local, but `SERVER_URL` gets used as part of the server record in the auth-server.

Reply:
Gotcha, I wasn't really thinking about mobile. I think since it's possible to develop just servers/servers & web/ servers & admin tools it might make more sense to default these to localhost, then just have docs somewhere explaining the ngrok/mobile setup.

Reply:
Wait a minute actually in terms of this registerServer script I don't see why mobile matters at all. Even if you're running them via ngrok I think using localhost would still work

Reply:
Huh, that actually makes sense. I thought I tried earlier and got a bunch of errors. Will double check.